### DISCUSSION TITLE
Membership not showing (correctly) on user profile

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#385] [General] [answered]
It appears that a membership isn't always showing on the "user info card". And I can't quite figure out why. I have added two screenshot of the same user: one where his membership is visible and one where it is missing.

In this instance I found the user by clicking on his avatar in the Digital Twin commnunity: ![Screenshot 2021-07-28 14 57 22](https://user-images.githubusercontent.com/87815729/127325861-fde9ce73-be0a-49fc-91ea-284dae7759c5.png)

In this instance I found the user via the search function: ![Screenshot 2021-07-28 14 18 10](https://user-images.githubusercontent.com/87815729/127325678-86ffd8b5-35f2-48d2-85f0-a16412fe5f19.png)

**Comment by FloriSpoelstra** [comment]
Another - but slightly different - one in this category. I now have a user that is showing incorrect memberships on his user profile card. This guy is not a member of the Chat- en Spraaktechnologie challenge, nor the Voice ID opportunity, but they do show up on his card. 

It appears to have a relation with the order in which I am viewing user profiles. So every now and then I see the memberships of the user I have previous viewed. It is not consistently going wrong so it might be hard to simulate for you what is happening here. Hope this is enough info to figure it out :)  

![Screenshot 2021-07-28 16 38 48](https://user-images.githubusercontent.com/87815729/127342112-9a8f2e44-d469-411f-adaf-63b5d189d42e.png)

**Reply by techsmyth** [reply]
Hmm, not a nice bug. Thanks for tracking down. Will raise on dev backlog & track there...

**Reply by techsmyth** [reply]
Being tracked via the following bug: https://github.com/alkem-io/client-web/issues/907

**Comment by techsmyth** [comment, marked as answer]
As noted above, there appears to be a bug in the client whereby state from a previous display is being re-used for a later display. Tracked under the following bug: alkem-io/client-web#907
